### ISSUE TITLE
Check and change default aria-label values

### ISSUE THREAD

**Issue by Januszpl** [SnowdogApps/magento2-alpaca-components#310]
TO prevent "missing to change" value for aria-label in some elements I would change aria-label values in components for more accurate text for ex.
"Slide to previous item" or "Previous slide" from "button arrow ....".

**Comment by Aekal** [comment]
It's done

**Comment by szafran89** [comment]
thx @Aekal
